config.rb

```diff
@@ -9,6 +9,15 @@
 
 after_build do |builder|
   begin
+    # Newly created files on feature branches do not yet exist in production, so tell HTML Proofer to ignore links to them
+    # Based on https://github.com/gjtorikian/html-proofer#ignoring-new-files
+    merge_base = `git merge-base main HEAD`
+    new_urls = %x(git diff -z --name-only --diff-filter=AC #{merge_base})
+      .split("\0")
+      .filter { |path| path.start_with? "source/" }
+      .map { |path| path.delete_prefix("source/").delete_suffix(".erb").delete_suffix(".md") }
```

Comment:
Could we do
```suggestion
      .map { |path| path.delete_prefix("source/").delete_suffix(".erb.md") }
```

Reply:
`delete_suffix(".erb").delete_suffix(".md")` correctly handles both `"foo.md.erb"` and `"foo.md"` (as well as e.g. `"foo.txt.erb"`)